### DISCUSSION TITLE
freeplane nas nuvens

### DISCUSSION THREAD

**Discussion by crmeier09** [freeplane/freeplane#1434] [Ideas]
1 - Eu não conheço freeplane nas nuvens.
Tem algum projeto em andamento?
2 - Freeplane para android / smastphone

**Comment by euu2021** [comment]
> 1 - Eu não conheço freeplane nas nuvens.
Tem algum projeto em andamento?

"Nas nuvens" você quer dizer colaboração simultânea? Isso já foi discutido algumas vezes. A última proposta que eu lembro de ter visto foi essa do nnako: https://github.com/freeplane/freeplane/discussions/426

> 2 - Freeplane para android / smastphone

Para Android, há um visualizador de mapas, do bekoepel: https://github.com/bekoeppel/droidplane
É só um visualizador, não dá para editar o mapa. Também não há uma opção de pesquisa, o que limita bastante o uso para mapas maiores.

**Comment by crmeier09** [comment]
ok


*Atenciosamente*,

*Celio Raul Meier*
=======================================
CURITIBA/PR – BRASIL


Em seg., 2 de out. de 2023 às 14:22, euu ***@***.***>
escreveu:
